bin/_mkasset.bat

```diff
@@ -16,6 +16,8 @@ set MINDBENDER_ASSETPATH=%2%MINDBENDER_ASSET%
 
 If not exist %MINDBENDER_ASSETPATH% (
 	mkdir %MINDBENDER_ASSETPATH%
+	mkdir %MINDBENDER_ASSETPATH%\work
+	mkdir %MINDBENDER_ASSETPATH%\publish
```

Comment:
There is no need to create the publish folder. It is automatically created when necessary.